### DISCUSSION TITLE
Transfer files from aws s3 to local

### DISCUSSION THREAD

**Discussion by gaganbiswas** [pocketbase/pocketbase#4913] [Q&A] [answered]
I had previously set up my pocketbase server to save the images/files to S3. Is there any way to download them locally and continue using them. I don't want to save files to S3 anymore and want those files to be saved locally.

**Comment by ganigeorgiev** [comment, marked as answer]
You can use external tools like [rclone](https://github.com/rclone/rclone) or [s5cmd](https://github.com/peak/s5cmd) to download your files in `pb_data/storage` and then switch the toggle from the Admin UI storage settings.

**Reply by gaganbiswas** [reply]
Okay, so I will be just downloading my s3 bucket and just upload the data into `pb_data/storage/` and it will then use the local directory storage for retrieving the images/files right?
